docker/cmsweb/Dockerfile

```diff
@@ -58,19 +62,19 @@ RUN mkdir -p $WDIR/bin
 
 # download node exporter
 WORKDIR ${WDIR}
-RUN curl -k -L -O https://github.com/prometheus/node_exporter/releases/download/v1.1.2/node_exporter-1.1.2.linux-amd64.tar.gz && tar xfz node_exporter-1.1.2.linux-amd64.tar.gz && cp node_exporter*/node_exporter $WDIR/bin && rm -r node_exporter-1.1.2.linux-amd64/ && rm -r node_exporter-1.1.2.linux-amd64.tar.gz
+#RUN curl -k -L -O https://github.com/prometheus/node_exporter/releases/download/v1.1.2/node_exporter-1.1.2.linux-amd64.tar.gz && tar xfz node_exporter-1.1.2.linux-amd64.tar.gz && cp node_exporter*/node_exporter $WDIR/bin && rm -r node_exporter-1.1.2.linux-amd64/ && rm -r node_exporter-1.1.2.linux-amd64.tar.gz
 
 # build exporters
-RUN git clone https://github.com/dmwm/cmsweb-exporters.git
-ARG CGO_ENABLED=0
-RUN cd cmsweb-exporters && \
-    go build -o $WDIR/bin/das2go_exporter -ldflags="-s -w -extldflags -static" das2go_exporter.go && \
-    go build -o $WDIR/bin/reqmgr_exporter -ldflags="-s -w -extldflags -static" reqmgr_exporter.go && \
-    go build -o $WDIR/bin/wmcore_exporter -ldflags="-s -w -extldflags -static" wmcore_exporter.go && \
-    go build -o $WDIR/bin/http_exporter -ldflags="-s -w -extldflags -static" http_exporter.go && \
-    go build -o $WDIR/bin/process_exporter -ldflags="-s -w -extldflags -static" process_exporter.go && \
-    go build -o $WDIR/bin/cpy_exporter -ldflags="-s -w -extldflags -static" cpy_exporter.go && \
-    go build -o $WDIR/bin/cmsweb-ping -ldflags="-s -w -extldflags -static" cmsweb-ping.go
+#RUN git clone https://github.com/dmwm/cmsweb-exporters.git
+#ARG CGO_ENABLED=0
+#RUN cd cmsweb-exporters && \
+#    go build -o $WDIR/bin/das2go_exporter -ldflags="-s -w -extldflags -static" das2go_exporter.go && \
+#    go build -o $WDIR/bin/reqmgr_exporter -ldflags="-s -w -extldflags -static" reqmgr_exporter.go && \
+#    go build -o $WDIR/bin/wmcore_exporter -ldflags="-s -w -extldflags -static" wmcore_exporter.go && \
+#    go build -o $WDIR/bin/http_exporter -ldflags="-s -w -extldflags -static" http_exporter.go && \
+#    go build -o $WDIR/bin/process_exporter -ldflags="-s -w -extldflags -static" process_exporter.go && \
+#    go build -o $WDIR/bin/cpy_exporter -ldflags="-s -w -extldflags -static" cpy_exporter.go && \
+#    go build -o $WDIR/bin/cmsweb-ping -ldflags="-s -w -extldflags -static" cmsweb-ping.go
 
 ENV PATH="${GOPATH}/src/github.com/vkuznet/cmsweb-exporters:${GOPATH}/src/github.com/vkuznet/cmsweb-exporters/bin:${PATH}"
```

Comment:
you don't need this anymore

Reply:
ok. I have commented all relevant lines for `GOPATH`. Thanks